coverage.sh

```diff
@@ -35,6 +35,10 @@ LLVM_PROFILE_FILE="${PROFRAW_FILE}" ${UNIT_TESTS_EXEC} || error_exit "Unit tests
 
 echo -e "\nRunning functional tests with coverage instrumentation..."
 (cd ${FUNCTIONAL_TESTS_DIR} && LLVM_PROFILE_FILE="${PROFRAW_FILE}" ./${FUNCTIONAL_TESTS_EXEC}) || error_exit "Functional tests execution failed."
+
+echo -e "\nRunning glu-demangle tests..."
+(cd tools/glu-demangle && cargo test) || error_exit "glu-demangle tests execution failed."
```

Comment:
The tool itself is not built anywhere, add it to the cmake maybe? I'd want a binary in `build/tools/glu-demangle`